src/core/Akka.Remote/MessageSerializer.cs

```diff
@@ -25,12 +25,10 @@ public static class MessageSerializer
         /// <returns>System.Object.</returns>
         public static object Deserialize(ActorSystem system, SerializedMessage messageProtocol)
         {
-            Type type = messageProtocol.HasMessageManifest
-                ? Type.GetType(messageProtocol.MessageManifest.ToStringUtf8())
-                : null;
-            var message = system.Serialization.Deserialize(messageProtocol.Message.ToByteArray(),
-                messageProtocol.SerializerId, type);
-            return message;
+            return system.Serialization.Deserialize(
```

Comment:
Initially, SerializerWithStringManifest was designed for Persistence, and used there everywhere. But right now Akka team uses this manifest serializer everywhere, especially in Cluster.Tools and Sharding